openlibrary/catalog/add_book/tests/test_add_book.py

```diff
@@ -991,9 +977,11 @@ def test_find_match_is_used_when_looking_for_edition_matches(mock_site) -> None:
     This also indirectly tests `merge_marc.editions_match()` (even though it's
     not a MARC record.
     """
+    # Unfortunately this Work level author is totally irrelevant to the matching
```

Comment:
Is this just a test issue? Do live Edition objects inherit authors from their Works?